### ISSUE TITLE
cheat toggle "enter" hotkey doesn't work

### ISSUE THREAD

**Issue by sumerhays** [TASVideos/BizHawk#1354]
i am using cheats for sonic 3 and knuckles and i need to be able to toggle individual cheats on and off without using a mouse, but the hotkey "enter" doesn't work.  there is an easy way to fix this right?

**Comment by alyosha-tas** [comment]
I tried adding in the shortcut key with:

this.ToggleMenuItem.ShortcutKeys = System.Windows.Forms.Keys.Enter;

but it throws this weird exception when opening the cheat menu:

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ComponentModel.InvalidEnumArgumentException: The value of argument 'value' (13) is invalid for Enum type 'Keys'.
Parameter name: value
   at System.Windows.Forms.ToolStripMenuItem.set_ShortcutKeys(Keys value)

I don't know why this is happening. I tried some different keys and the only one that doesn't throw an exception that I found so far is num lock.
